### PR TITLE
bug fix change chat after sending message

### PR DESCRIPTION
**Pull request by Vuurvos1**
fixed a bug where you would be sent to the wrong chat after having sent a message in another chat, and a bug where you could send HTML elements in your own chat.

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ router.get('/users', async (req, res) => {
...
+      .collection('chats')
+      .find({
+        participants: {
+          $in: [id],
+        },
+      })
+      .toArray();
 
   const x = [];
   for (const i of chats) {
```
Comment (SimonPlanje) [inline routes/users.js:22]: Is this one being used? Because the function is empty

File: routes/likepage.js [hunk 2]
```diff
@@ -1,74 +1,71 @@
...
     // console.log(users);
 
     const index = await db
-      .get()
-      .collection('users')
-      .findOne({ _id: ObjectId(req.session.user._id) });
+        .get()
+        .collection('users')
+        .findOne({_id: ObjectId(req.session.user._id)});
     // console.log('1 GEBRUIKER LADEN');
```
Comment (SimonPlanje) [inline routes/likepage.js:20]: This is not your code, but we probably should delete these logs

File: routes/users.js (hunk 1 above)
Comment (Vuurvos1) [reply routes/users.js:22]: Yes it is being used, it belongs to the code above

**Comment by Linux-localhost** [comment]
strijder @Vuurvos1

**Pull request merged**